titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/JobManagerConfiguration.java

```diff
@@ -37,6 +39,13 @@
     @DefaultValue("300")
     int getActiveNotStartedTasksLimit();
 
+    /**
+     * Maximum number of concurrent store updates per job within the reconciliation loop. Currently only task kill
+     * operations are accounted for when a job moves to {@link JobState#KillInitiated} state.
```

Comment:
add a note that big scale downs (`updateJobCapacity()`) will also not be accounted?

Reply:
I think we have to throttle them as well. Let me add it.